language-support/ts/daml-ledger/index.ts

```diff
@@ -997,6 +997,14 @@ class Ledger {
     const ledgerResponse = jtv.Result.withException(
       decodeLedgerResponse.run(json),
     );
+    if (!(ledgerResponse.status >= 200 && ledgerResponse.status <= 299)) {
+      console.error(
```

Comment:
looking at how we handle `throwOnError` above, this should probably be `console.log` not `console.error`. No reason to treat this any differently.